zgw/besluiten-api/src/main/kotlin/com/ritense/besluitenapi/BesluitenApiPlugin.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.besluitenapi
+
+import com.ritense.plugin.annotation.Plugin
+import com.ritense.plugin.annotation.PluginProperty
+import com.ritense.zgw.Rsin
+import mu.KLogger
+import mu.KotlinLogging
+import java.net.URI
+
+@Plugin(key = BesluitenApiPlugin.PLUGIN_KEY,
+    title = "Besluiten API",
+    description = "Connects to the Besluiten API")
+class BesluitenApiPlugin {
```

Comment:
missing configuration name property 

The new 'Besluiten API' plugin can be configured with the properties:
Configuration name
URL (String)

RSIN (String)

Authentication plugin configuration